.github/workflows/seqera-showcase-staging.yml

```diff
@@ -116,7 +116,7 @@ jobs:
             compute-envs/staging*.yaml \
             ${{ ( github.event_name != 'workflow_dispatch' || inputs.pre_run ) && format('--pre_run "{0}"', inputs.pre_run) || '' }} \
             ${{ ( github.event_name != 'workflow_dispatch' || inputs.config ) && format('--config "{0}"', inputs.config) || '' }} \
-            ${{ ( github.event_name != 'workflow_dispatch' || inputs.launch_container ) && format('--launch-container "{0}"', inputs.launch_container) || '' }} \
+            ${{ ( github.event_name != 'workflow_dispatch' || inputs.launch_container ) && format('--launch_container "{0}"', inputs.launch_container) || '' }} \
```

Comment:
Why `--launch_container`? the cli option is  `--launch-container` 

https://github.com/seqeralabs/tower-cli/pull/444/files#diff-be6460a3f7b7a601bef11ba689bbfe2a98b81d1a1acd5d631e790f92e681cc3cR107

Reply:
I see, and how it knows to use `--launch-container` when invoking `tw` cli?

Reply:
Got it!